src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanContext.hpp

```diff
@@ -1468,6 +1468,13 @@ struct TScanContext final
         return m_messageType;
     }
 
+    /**
+     * @brief Feed source information.
+     *
+     * @return Pair with ADP base name, and ADP expanded description.
```

Comment:
Improve: "and ADP expanded description" This doesn't seem right

```suggestion
     * @return Pair with CNA base name and CNA expanded name.
```